appveyor.yml

```diff
@@ -0,0 +1,28 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: C:\projects\asset-lib
+branches:
+  except:
+    - gh-pages
+
+install:
+  - cinst -y OpenSSL.Light
+  - SET PATH=C:\Program Files\OpenSSL;%PATH%
+  - sc config wuauserv start= auto
+  - net start wuauserv
+  - cinst -y php
```

Comment:
I'm afraid this might break when newer php versions will come out. Would be awesome if we can tell `^7.1`.

Found some docs here
https://github.com/chocolatey/choco/wiki/CommandsInstall#examples